README.md

```diff
@@ -66,9 +82,10 @@ yarn install
 yarn start
 ```
 
-This should automatically open your browser with [localhost:8000/substrate-front-end-template].
+This should automatically open your browser with [0.0.0.0:8000/substrate-front-end-template].
```

Comment:
Does not work for me. I get a page with an error.
```
Error Connecting to Substrate
{ "isTrusted": true }
```
Seems like the frontend cannot connect to the node. Any idea what could be the problem?

Another thought: Should we drop the frontend? Why did you include it in the first place? It is probably not very useful as we cannot interact with it without additional go-perun tooling.

Reply:
In development.json, change PROVIDER_SOCKET to "ws://localhost:9944".